pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConfiguration.java

```diff
@@ -412,6 +412,14 @@
     private String tlsTrustStorePassword = null;
 
     /**** --- KeyStore TLS config variables used for proxy to auth with broker--- ****/
+    @FieldContext(
+            category = CATEGORY_KEYSTORE_TLS,
+            dynamic = true,
+            doc = "Enable TLS when talking with other brokers in the same cluster (admin operation)"
+                    + " or different clusters (replication)"
```

Comment:
Looks the doc is from the broker.conf? We need a more suitable description

Reply:
ok.